tests/components/ozw/test_light.py

```diff
@@ -316,6 +316,39 @@ async def test_light(hass, light_data, light_msg, light_rgb_msg, sent_messages):
     assert state.state == "on"
     assert state.attributes["color_temp"] == 465
 
+    # Test setting invalid color temp
+    new_color = 120
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "color_temp": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 19
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#00000000ff", "ValueIDKey": 659341335}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)